src/components/Editor/index.js

```diff
@@ -646,14 +647,32 @@ class Editor extends Component {
       }));
   }
 
-  editorHeight() {
-    const { selectedSource, horizontal } = this.props;
+  getInlineEditorStyles() {
+    const { selectedSource, horizontal, searchOn } = this.props;
 
-    if (!shouldShowFooter(selectedSource, horizontal)) {
-      return "100%";
+    if (shouldShowFooter(selectedSource, horizontal)) {
+      // The footer is on and so we need to return different values due to the
+      // size of the footer to make sure we have nothing being pushed off the
+      // screen.
+      if (searchOn) {
+        return {
+          height: "calc(100% - var(--editor-searchbar-height)" +
```

Comment:
It is so much cleaner than the other ways we explored. I'd also recommend making that change as Jason mentioned. Other than that looks good to me.